proposals/relaxed-simd/Overview.md

```diff
@@ -37,6 +37,77 @@ Example of some instructions we would like to add:
 - Relaxed Swizzle (implementation defined out of bounds behavior)
 - Relaxed Rounding Q-format Multiplication (optional saturation)
 
+### Consistency
+
+This proposal introduces non-deterministic instructions - given the same
```

Comment:
I think "non-deterministic instructions" here (and one more below) might have to change to something like "hardware-dependent instructions", because we have the section below about how the non-determinism is encapsuated in fpenv, making instructions deterministic. It can be a bit confusing.